.github/workflows/drc_sc9_test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
`sc9` should be the proper name for the library.

Reply:
Noted.